rero_ils/modules/item_types/api.py

```diff
@@ -75,7 +75,7 @@ def extended_validation(self, **kwargs):
         """
         online_type_pid = self.get_organisation().online_circulation_category()
         if self.get('type') == 'online' and online_type_pid and \
-                self.pid != online_type_pid:
+           self.pid != online_type_pid:
```

Comment:
This is not recommend in pep8:
`E125 continuation line with same indent as next logical line`